css/styles.css

```diff
@@ -0,0 +1,137 @@
+body {
+    margin: 0;
+}
+.topnav {
+    background-color:rgb(25, 50, 126);
+    overflow: hidden;
+    width: 100%;
+    height: 3em;
+}
+
+.topnav a {
+    float: left;
+    color: whitesmoke;
+    text-align: center;
+    padding-top: 1vh;
+    padding-right: 2vw;
+    text-decoration: none;
+    font-size: 1.2em;
+    text-shadow: 2px 2px black;
+}
+.topnav img {
+    float: left;
+    margin-top: 0.4vh;
+    margin-right: 0.5vw;
+    margin-left: 0.2vw;
```

Comment:
instead of 3 rows of margin top, right and left - use short version of margin:
{
  margin: a,b,c,d;
}